packages/app/src/app/utils/inject-resources-for-export.ts

```diff
@@ -0,0 +1,42 @@
+import { Module } from '@codesandbox/common/lib/types';
+import {
+  resourceIsCss,
+  createExternalCSSLink,
+  createExternalJSLink,
+} from 'sandbox/external-resources';
+
+const PATHS_TO_INJECT = ['/public/index.html'];
```

Comment:
This will work in react but not in vanilla for example since that one is at `./index.html` maybe the best way is to get all the .HTML files and inject there no?

From the modules maybe?

Reply:
yep, that makes more sense, I'll make it modify all the `*.html` then 👍